test/starlark_tests/macos_application_tests.bzl

```diff
@@ -445,6 +445,16 @@ def macos_application_test_suite(name):
         tags = [name],
     )
 
+    archive_contents_test(
```

Comment:
Could we add another one of these for ios_application

Reply:
I don't think .dylibs are allowed to be packaged in the same way for ios_applications. If this is desired can any reviewers with experience in this area advise on how this should work?

Reply:
Ah yeah I believe they'd need to be packaged as frameworks.